app/globals.css

```diff
@@ -72,6 +72,16 @@ h4 {
   --dark-background: #111827;
   --light-foreground: #020202;
   --dark-foreground: #f9fafb;
+  --primary-50: #f8f5fc;
+  --primary-100: #f0eafb;
+  --primary-200: #d3bff3;
+  --primary-300: #b08ee6;
+  --primary-400: #976ae2;
+  --primary-500: #6025c0;
+  --primary-600: #4d1e9a;
+  --primary-700: #380d80;
+  --primary-800: #35156b;
+  --primary-900: #1f0d40;
```

Comment:
Merge with latest main as that imports `theme.css` which contains these css variables. Avoid redefining the CSS variables